radicle-httpd/src/api/v1.rs

```diff
@@ -18,3 +18,21 @@ pub fn router(ctx: Context) -> Router {
 
     Router::new().nest("/v1", routes)
 }
+
+#[cfg(test)]
+mod utils {
```

Comment:
right place/name?

Reply:
Name is fine, but maybe should go one module up, since it's not related to `v1`, ie. in `api.rs`

Reply:
I think this is the right place, because in future we might have a `v2` and it should have its own tests. Having them in `api.rs` would be confusing.

Reply:
moved it all to another file, `radicle-httpd/src/api/test.rs`